app/assets/javascripts/quick_create.js

```diff
@@ -0,0 +1,93 @@
+function addToAutocomplete(json) {
+  var source = JSON.parse($("#" + json.autocomplete_id).attr("data-autocomplete-source"));
+  source.unshift(json.name)
+  $("#" + json.autocomplete_id).attr("data-autocomplete-source", JSON.stringify(source));
+  var sourceIds = JSON.parse($("#" + json.autocomplete_id).attr("data-autocomplete-source-ids"));
+  sourceIds.unshift(json.id);
```

Comment:
... vice versa, if we don't need them, would it be better to centralize it in the backend?